projects/cloud/app/graphql/types/query_type.rb

```diff
@@ -39,7 +39,7 @@ def accounts
       argument :account_name, String, required: true
     end
     def project(name:, account_name:)
-      ProjectFetchService.call(name: name, account_name: account_name)
+      ProjectFetchService.call(name: name, account_name: account_name, current_user: context[:current_user])
```

Comment:
Nitpick: I think the name of the parameter should be `user:` instead of `current:`. While `current` makes sense in the context of a controller to represent the authenticated user, I think the service doesn't have that context. Give me the user and I'll give you the projects the user has access to.